charts/pulsar/templates/zookeeper/zookeeper-statefulset.yaml

```diff
@@ -41,7 +41,7 @@ spec:
       component: {{ .Values.zookeeper.component }}
   updateStrategy:
     type: {{ .Values.zookeeper.updateStrategy }}
-  podManagementPolicy: {{ .Values.zookeeper.podManagementPolicy | default "OrderedReady" }}
+  podManagementPolicy: {{ .Values.zookeeper.podManagementPolicy | default "Parallel" }}
```

Comment:
why do we need to change the default from OrderedReady to Parallel?

Reply:
I remember that's a suggstion from Yang Yang, if using pulsar operators this is not required for zk. But native statefulset ZK seems suggest to use Parallel, also refer the https://github.com/apache/pulsar-helm-chart/blob/master/charts/pulsar/values.yaml#L287